### ISSUE TITLE
error: converting to 'std::tuple<int, int, int>'

### ISSUE THREAD

**Issue by sagitter** [mkeeter/antimony#112]
Hi all,

```antimony 0.9.0``` build stops with this error:

```
../../../lib/fab/inc/fab/types/shape.h:19:50: error: converting to 'std::tuple<int, int, int>' from initializer list would use explicit constructor 'constexpr std::tuple< <template-parameter-1-1> >::tuple(_UElements&& ...) [with _UElements = {int, int, int}; <template-parameter-2-2> = void; _Elements = {int, int, int}]'
           std::tuple<int,int,int> color={-1,-1,-1});
```

gcc (GCC) 5.1.1 20150618 (Red Hat 5.1.1-4) on Fedora 22 64bit.
Full log: http://fpaste.org/252268/

Why does happen?
Thanks.

**Comment by mkeeter** [comment]
Yes: there's an issue with type inference in gcc.  It was fixed in 42b49fbd0a1714f118e63107103d435fadb561bc but I didn't merge that into master; folks that are building their own Antimony are encouraged to work from the `develop` branch.

Can you update to the latest `develop` commit and see if that solves your problem?

**Comment by sagitter** [comment]
Thanks.
